src/providers/provider.ts

```diff
@@ -26,6 +26,7 @@ type BlockHash = string;
 type BlockHeight = number;
 export type BlockId = BlockHash | BlockHeight;
 
+// TODO: Remove near-final?
```

Comment:
`near-final` used to be in the docs, but I don't see it anymore. Maybe this was removed. Who can we ask?

Seems like we could make a separate PR that just removes this option, and request review from people who would know enough to approve or reject the change.

Reply:
No idea, that's why I just added TODO

Reply:
Seems like this is secret feature 🤷‍♂️